.github/workflows/r-cmd-check.yaml

```diff
@@ -0,0 +1,35 @@
+on: [push, pull_request]
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/setup-r@master
+      - uses: r-lib/actions/setup-pandoc@master
+      - name: Query dependencies
+        run: |
+          install.packages(c('remotes', 'rcmdcheck', 'covr', 'rmarkdown'))
```

Comment:
Two questions:

1. Can these packages be cached so they don't need to be installed every time?

1. Are there any obvious advantages of using the **rcmdcheck** package compared to the simple command line `R CMD check`? I have to admit that I'm not familiar with this package.